frontend/src/components/FAQ/FAQNavBar.tsx

```diff
@@ -41,7 +42,7 @@ const FAQNavBar = (): ReactElement => {
 
   const displayNavBar = (): ReactElement => {
     return (
-      <ul className="nav nav-button-margin">
+      <ul className={'nav ' + styles.navButtonMargin}>
```

Comment:
Try using [ES6 template strings](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here instead of just string concatenation